src/services/CicService.ts

```diff
@@ -190,6 +193,10 @@ export class CicService {
 				HttpCodesEnum.SERVER_ERROR,
 			);
 		}
+		
+		if (sessionItem.Items[0].expiryDate < absoluteTimeNow()) {
+			throw new AppError(`Session with session id: ${sessionItem.Items[0].sessionId} has expired`, HttpCodesEnum.UNAUTHORIZED);
+		}
```

Comment:
This isn't working @Caitlin-cooling 
<img width="1427" alt="image" src="https://github.com/alphagov/di-ipv-cri-cic-api/assets/40401118/3abaee0a-951f-4842-b1d3-50582567d76e">


<img width="1885" alt="image" src="https://github.com/alphagov/di-ipv-cri-cic-api/assets/40401118/d1afded2-7a7e-4a53-830f-1e021881df0b">